### PR TITLE
LC 2473 restrict org by domain

### PR DESCRIPTION
**Pull request by jaylco**
- Include new orgs V2 endpoint
- Add Domains to organisational unit object
- Filter organisational unit dropdown by domain (for all users)
- Cache enhancements

### CODE REVIEW

File: src/lib/model.ts [hunk 1]
```diff
@@ -884,6 +881,10 @@ export class OrganisationalUnit {
 		return exists
 	}
 
+	doesDomainExist(domain: string) {
+		return this.domains.find(d => d.domain === domain) !== undefined
+	}
+
 	formatNameWithAbbrev() {
```
Comment (SavvasCO) [inline src/lib/model.ts:888]: Assign a string return type to this function

**Review by SavvasCO — COMMENTED** [review]
Looks great! Some minor comments.

File: src/lib/model.ts [hunk 2]
```diff
@@ -884,6 +881,10 @@ export class OrganisationalUnit {
 		return exists
 	}
 
+	doesDomainExist(domain: string) {
```
[the remark this answers is not in the document]
Comment (jaylco) [reply src/lib/model.ts:884]: Either or works to be honest - it's to match the existing `doesDomainExistInToken` method above it.

**Review by SavvasCO — APPROVED** [review]
LGTM!

**Pull request merged**